src/course-unit/data/api.js

```diff
@@ -148,3 +149,31 @@ export async function duplicateUnitItem(itemId, XBlockId) {
 
   return data;
 }
+
+/**
+ * Get an object containing course outline data.
+ * @param {string} courseId - The identifier of the course.
+ * @returns {Promise<Object>}
```

Comment:
Would it be possible for you to specify the actual shape of the data returned, instead of just `Object` ?

Reply:
Added

Reply:
Nice, thank you!